kassy

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# kassy - A bash CLI tool for running kassy in docker
+# Written By: Matthew Hartstonge <matt@mykro.co.nz>
+
+# Soak in user params
+ARGS="$@"
+
+function usage() {
+  echo "usage: kassy [--all] [--fb] [--skype] [--slack] [--test]               "
```

Comment:
Except there are cases where we want to put stuff in the config file but one test with one of the integrations at a time